src/styles/Utils/mixins.scss

```diff
@@ -0,0 +1,14 @@
+@mixin centererd-hands() {
+  position: absolute;
+  bottom: 40vmin;
+}
```

Comment:
The mixin name `centererd-hands` seems to be a typo. According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), names should be spelled correctly. Consider renaming it to `centered-hands`.